### PR TITLE
Handle nulls in new boxes

### PR DESCRIPTION
**Pull request by apmasell**

### CODE REVIEW

File: spring/src/main/java/uk/ac/bbsrc/tgac/miso/spring/LimsBindingInitializer.java [hunk 1]
```diff
@@ -531,6 +537,32 @@ public void setAsText(String element) throws IllegalArgumentException {
       }
 
     });
+    binder.registerCustomEditor(BoxUse.class, new PropertyEditorSupport() {
+      @Override
+      public void setAsText(String element) throws IllegalArgumentException {
+        long id = Long.parseLong(element);
+        try {
+          setValue(sqlBoxDAO.getUseById(id));
+        } catch (IOException e) {
```
Comment (djcooke) [inline spring/src/main/java/uk/ac/bbsrc/tgac/miso/spring/LimsBindingInitializer.java]: Should this not be thrown? Seems the error will be hidden and bad boxes sent to the service/requestManager layer.

Same with below for BoxSize

Comment (callunity) [inline spring/src/main/java/uk/ac/bbsrc/tgac/miso/spring/LimsBindingInitializer.java]: It's now thrown.

**Comment by djcooke** [comment]
:+1:

**Comment by callunity** [comment]
:+1:

**Pull request merged**